crates/matrix-sdk/src/event_cache/pagination.rs

```diff
@@ -281,13 +281,7 @@ impl RoomPagination {
             .state
             .write()
             .await
-            .handle_backpagination(
-                response.chunk,
-                response.end,
-                prev_token,
-                #[cfg(feature = "experimental-search")]
-                &room,
-            )
+            .handle_backpagination(response.chunk, response.end, prev_token)
```

Comment:
Sweet! Can you also revert to the previous version of the code, which kept the `Room` instance in a smaller sub-scope, around the pagination itself?